cmd/certificate-generator/main.go

```diff
@@ -0,0 +1,32 @@
+/*
```

Comment:
Making certficate-generator as a separate bin file is not necessary, refreshing a certificate can run in the background goroutine.

Reply:
@MegaByte875 Got it, do you mean I should run this as a background goroutine in the controller-manager container? I also use this script to generate the initial cert in an init-container during pod start up to make sure the controller-manager and autoscaler container don't crash during start-up because of an invalid certificate. Is is possible to also use a background goroutine for this?
https://github.com/vesoft-inc/nebula-operator/blob/autoscaler-webhook/charts/nebula-operator/templates/controller-manager-deployment.yaml#L36-L52

Reply:
Fixed